src/Vault.sol

```diff
@@ -618,7 +777,32 @@ contract Vault is ERC4626, ERC20Permit, ILiquidationSource, Ownable {
    * @dev `_from` must have a balance of at least `_shares`.
    */
   function _transfer(address _from, address _to, uint256 _shares) internal virtual override {
-    _twabController.twabTransfer(_from, _to, uint112(convertToAssets(_shares)));
-    super._transfer(_from, _to, _shares);
+    _twabController.twabTransfer(_from, _to, uint112(_shares));
+  }
+
+  /**
+   * @notice Set claimer address.
+   * @param claimer_ Address of the claimer
+   */
+  function _setClaimer(Claimer claimer_) internal {
+    _claimer = claimer_;
+  }
+
+  /**
+   * @notice Set yield fee percentage.
+   * @dev Yield fee is represented in 9 decimals and can't exceed `1e9`.
+   * @param yieldFeePercentage_ Yield fee percentage
+   */
+  function _setYieldFeePercentage(uint256 yieldFeePercentage_) internal {
+    require(yieldFeePercentage_ <= FEE_PRECISION, "Vault/yieldFeePercentage-gt-1e9");
+    _yieldFeePercentage = yieldFeePercentage_;
+  }
+
+  /**
+   * @notice Set yield fee recipient address.
+   * @param yieldFeeRecipient_ Address of the fee recipient
+   */
+  function _setYieldFeeRecipient(address yieldFeeRecipient_) internal {
+    _yieldFeeRecipient = yieldFeeRecipient_;
```

Comment:
Do we need this function?

Reply:
Yes cause we use it in the constructor and in the `setYieldFeeRecipient` function.
https://github.com/pooltogether/v5-prize-vault/blob/3e9268fc221769398ae97eb26e4b5239eb130fce/src/Vault.sol#L177
https://github.com/pooltogether/v5-prize-vault/blob/3e9268fc221769398ae97eb26e4b5239eb130fce/src/Vault.sol#L497